delocate/libsana.py

```diff
@@ -561,7 +577,7 @@ def resolve_rpath(lib_path, rpaths):
         return lib_path
 
     lib_rpath = lib_path.split("/", 1)[1]
-    for rpath in rpaths:
+    for rpath in tuple(rpaths) + _default_paths_to_search:
```

Comment:
If you're going to join them as a tuple here then you might as well unpack them together:
```suggestion
    for rpath in (*rpaths, *_default_paths_to_search):
```